pynapple/core/time_series.py

```diff
@@ -638,6 +638,51 @@ def interpolate(self, ts, ep=None, left=None, right=None):
 
         return _initialize_tsd_output(self, new_d, time_index=new_t, time_support=ep)
 
+    def gradient(self, ep=None):
+        """Compute the gradient of the time series using numpy.gradient.
```

Comment:
And then be super clear in the description that it computes the derivatives only in respect to time

Reply:
https://github.com/pynapple-org/pynapple/pull/451/commits/64946202de18f837247789d28f6c07ee519767c8